src/test/java/com/google/sps/UserUtilTest.java

```diff
@@ -386,4 +386,35 @@ public void noCookies(){
     
     Assert.assertEquals(expected, actual);
   }
-}
+
+  // addPoints where user already has some points
+  @Test
+  public void addPoints_additionalPoints(){
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity user1 = new Entity("user");
+    user1.setProperty("score", 10);
+    user1.setProperty("userId", "newUser1");
+    datastore.put(user1);
+
+    UserUtils.addPoints(user1, 20, datastore);
+
+    Entity userEntity = UserUtils.getEntityFromDatastore("user", "userId", "newUser1", datastore);
+    
+    Assert.assertEquals(30, (long) userEntity.getProperty("score"));
+  }
+
+  // addPoints where user doesn't have points
+  @Test
+  public void addPoints_firstPoints(){
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity user1 = new Entity("user");
+    user1.setProperty("userId", "newUser1");
+    datastore.put(user1);
+
+    UserUtils.addPoints(user1, 20, datastore);
+
+    Entity userEntity = UserUtils.getEntityFromDatastore("user", "userId", "newUser1", datastore);
+    
+    Assert.assertEquals(20, (long) userEntity.getProperty("score"));
```

Comment:
I think you should change this so that you don't have to cast a long value here, my comment above gives a suggestion on how you might be able to do this

Reply:
I still need to cast the "expected" to long to avoid errors, but otherwise it works